src/test/emmetHelper.test.ts

```diff
@@ -952,7 +957,7 @@ describe('Test completions', () => {
 		return updateExtensionsPath(null).then(() => {
 
 
-			const document = TextDocument.create('test://test/test.html', 'html', 0, 'lorem10.item');
```

Comment:
Umm actually I'm not sure about this. Probably lorem10.item doesn't work in newer emmet and the tests was meant to check only the number of words hence changed it.